core/src/main/java/io/grpc/internal/ClientTransport.java

```diff
@@ -108,12 +108,17 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
      * that this method is called without {@link #shutdown} being called.  If the argument to this
      * function is {@link Status#isOk}, it is safe to immediately reconnect.
      *
+     * <p>This is called at most once, and must be called prior to {@link #transportTerminated}.
```

Comment:
Clarified that both `transportShutdown()` and `transportTerminated()` are called exactly once.